testing/run_api_tests.sh

```diff
@@ -37,6 +36,33 @@ fi
 
 # Initialise test data
 case $1 in
+    postgres|mysql|mariadb)
+    docker compose -f "$script_dir/docker-compose.yml" down
+    docker volume rm "testing_wakapi-$1"
+
+    docker_down=1
+    docker compose -f "$script_dir/docker-compose.yml" up --wait -d "$1"
```

Comment:
I think this is supposed to read `docker-compose`, no?

Reply:
Looks like this won't be possible. 
https://github.com/muety/wakapi/actions/runs/3786548513/jobs/6437545137#step:4:111

Reply:
Here's an explanation of the new syntax:
https://stackoverflow.com/a/66516826

Reply:
Note to all Linux users: you'll need to [install `docker-compose-plugin`](https://docs.docker.com/compose/install/linux/#install-using-the-repository).